pal/src/cruntime/wchar.cpp

```diff
@@ -1241,14 +1241,21 @@ PAL_wmemcmp(
 
     if (string1 == string2) return diff;
 
-    for (i = 0; i < count; i++)
+    constexpr size_t blockSize = sizeof(size_t) / sizeof(char16_t);
+    const     size_t *num1     = (const size_t*)(string1);
+    const     size_t *num2     = (const size_t*)(string2);
+
+    while( (count >= blockSize * wi) && !(num1[wi] - num2[wi]) ) ++wi;
```

Comment:
Yep, a silly edit issue on my end. That's why the test fails on CI. Thanks for pointing it out though.